apps/dashboard/app/views/batch_connect/sessions/connections/_novnc.html.erb

```diff
@@ -6,10 +6,10 @@
 
  <div class="row">
   <div class="col-sm-6">
-   <%= f.number_field(:compression, class: 'custom-range', type: 'range', min: 0, max: 9, value: 6, label: "Compression", help: "0 (low) to 9 (high)") %>
+   <%= f.number_field(:compression, class: 'custom-range', type: 'range', min: 0, max: 9, value: Configuration.novnc_connection_compression, label: "Compression", help: "0 (low) to 9 (high)") %>
   </div>
   <div class="col-sm-6">
-   <%= f.number_field(:quality, class: 'custom-range', type: 'range', min: 0, max: 9, value: 2, label: "Image Quality", help: "0 (low) to 9 (high)") %>
+   <%= f.number_field(:quality, class: 'custom-range', type: 'range', min: 0, max: 9, value: Configuration.novnc_connection_quality, label: "Image Quality", help: "0 (low) to 9 (high)") %>
```

Comment:
I think we should cast `to_i` here just to be sure it's a number. (funny strings I think cast to 0 which is OK in case they configured something funny.)